src/muya/lib/contentState/updateCtrl.js

```diff
@@ -4,8 +4,8 @@ import { CLASS_OR_ID } from '../config'
 
 const INLINE_UPDATE_FRAGMENTS = [
   '^([*+-]\\s)', // Bullet list
-  '^(\\[[x\\s]{1}\\]\\s)', // Task list
-  '^(\\d+\\.\\s)', // Order list
+  '^(\\[[xX\\s]{1}\\]\\s)', // Task list
```

Comment:
We have use the `i` flag to match by case insensitive, but it doesn't matter.

Reply:
I think this is ok, marked.js does the same.